JeMPI_Apps/JeMPI_UI/src/DevTools.tsx

```diff
@@ -1,10 +1,10 @@
-import { config } from 'config'
 import { lazy } from 'react'
 
-export const ReactQueryDevtools = !config.isDev
-  ? () => <></>
-  : lazy(() =>
-      import('@tanstack/react-query-devtools').then(res => ({
-        default: res.ReactQueryDevtools
-      }))
-    )
+export const ReactQueryDevtools =
+  process.env.NODE_ENV !== 'development'
```

Comment:
So I do think this presents a security risk (i.e someone can open their browser console and type `process.env.NODE_ENV = 'development'` to get access to the react dev tool. I actually dont think we should be bundling any dev specify code when deplying to production. It is a security risk . I do think it's fine for now/this PR (as that a longer discussion), but we need to look at that soon.

Reply:
These changes are updated and that file is removed.
more information : 
https://tanstack.com/query/v4/docs/react/devtools